src/index.ts

```diff
@@ -26,13 +26,37 @@ const hasExtensionRegex = /\.\w+$/;
 const extensions = ['.ts', '.tsx'];
 const extensionsRegex = new RegExp(`\\${extensions.join('|\\')}`);
 
+let TSConfig: CompilerOptions;
+
 // Custom resolver to allow `.ts` and `.tsx` extensions, along with finding files if no extension is provided.
 export async function resolve(
   specifier: string,
   context: ResolveContext,
   defaultResolve: Function,
 ): Promise<ResolveResponse> {
-  const { parentURL = baseURL } = context;
+  let { parentURL = baseURL } = context;
```

Comment:
It's a bit of a hack, but I need to swap out the "/*" in the key and the path, and then force relative so that the fileFinder function runs since most people aren't using extension ESModule imports in Node.JS. And I need to resolve the path from the TSConfig path because of how the baseUrl and stuff is used in TypeScript. If I was the person writing the TypeScript side of the Paths feature I would have done it diferently.